README.md

```diff
@@ -260,6 +260,24 @@ Lets understand this with an example :
 You have 2 fragments namely ProfileFragment and HomeFragment, which extends from BaseFragment. BaseFragment has a function fetchPosts() in the onCreate() method, Now if in future you decide that ProfileFragment should not fetch posts when it is created, rather it should first show a dialogBox if the user is not logged-in. If the codebase is huge, you may have hardtime refactoring it. The other way around is to create a kotlin file as fun fetchPosts() and then use this function in your fragments in the onCreate() method or in a swipeRefresh() method (depending on your use case). Also one must note that a class can extend only one abstract base class.
 <br/-> Read more [here](https://codeburst.io/inheritance-is-evil-stop-using-it-6c4f1caf5117), [here](https://dev.to/antonholmberg/the-baseclass-anti-pattern-16il), [here](https://proandroiddev.com/say-no-to-baseactivity-and-basefragment-83b156ed8998).
        
+29) Create sourceSets for your main layout folder as follows
```

Comment:
Hey, very nice point. Thank you for raising the PR. COuld you please add an exmaple with old structure and then new structure on right side. Developers will be easy to differentiate how it creates the value. Thank you once again @aqua30 :)

Reply:
Hi @niharika2810 , 

Thank you for the review.   

I've updated the image with before and after comparison.